packages/create-svelte/bin.js

```diff
@@ -79,6 +80,18 @@ async function main() {
 				initial: false,
 				active: 'Yes',
 				inactive: 'No'
+			},
+			{
+				type: 'select',
+				name: 'adapter',
+				message: 'Add an adapter',
+				choices: [
+					{
+						title: "I'll add it later or use a community-provided adapter (https://bit.ly/3kvqIRy)",
```

Comment:
do we really want to drive traffic through bit.ly here? who controls that link.
Would rather want to see an npm keyword search or a link to a svelte community page

Reply:
I agree on not using bitly. The full link (https://sveltesociety.dev/components/#category-SvelteKit%20Adapters) will make the option longer but I think its still the better option. I copied the link from the related issue shared by Ben.

Reply:
https://sveltesociety.dev/components#category-SvelteKit%20Adapters seemed kind of long to me. Maybe we can updates sveltesociety.dev first to make it something shorter like https://sveltesociety.dev/components#adapters

Reply:
I moved the link to the description on next line .

![image](https://user-images.githubusercontent.com/44284655/134518243-0c83f88b-76c7-4162-8bf6-004245fa5ae0.png)

If that is not good enough, I can make the change on sveltesociety.dev.

Reply:
I think making the change on sveltesociety.dev would be really nice. It's a pretty long description right now